rplugin/python3/deoplete/sources/tabnine.py

```diff
@@ -154,19 +162,28 @@ def _restart(self):
         if self._proc is not None:
             self._proc.terminate()
             self._proc = None
+            self._selector.close()
+            self._selector = None
         binary_dir = os.path.join(self._install_dir, 'binaries')
         path = get_tabnine_path(binary_dir)
         if path is None:
             self.print_error('no TabNine binary found')
             return
         self._proc = subprocess.Popen(
-            [path, '--client', 'sublime', '--log-file-path',
-             os.path.join(self._install_dir, 'tabnine.log')],
+            [
+                path,
+                '--client', 'sublime',
```

Comment:
Could you also change the `--client` flag to be `deoplete`?

Reply:
Does it make any difference on the operation of TabNine?

Reply:
In any case this should be a different PR

Reply:
It helps us to spot bugs and their origin